### ISSUE TITLE
Any script available to train the model on a custom dataset?

### ISSUE THREAD

**Issue by ajinkya-kulkarni** [jono-m/OrganoID#9]
Hello team, thanks for the repo. Is there any script/notebook available to train the model on a custom dataset? Thanks!

**Comment by schmoogol** [comment]
Have a look at the Split.py, Augment.py and Train.py files in /CommandLine. These are what you can use to prepare a dataset and train your own model (e.g. python OrganoID.py split/augment/train...).

**Comment by ajinkya-kulkarni** [comment]
Ah okay, thanks! I already have split my dataset and Augmented it from my end, so I think `Train.py` will be the most helpful to me. And as I understand the code, `Train.py` requires the training data as images in `images` and the corresponding masks in `segmentations` right?

**Comment by schmoogol** [comment]
Correct, and you also need a validation dataset (/validation/images and /validation/segmentations).
